### PR TITLE
Default values for forms

### LINKED ISSUE #541: Default values for forms
When creating new tasks or checkers, some commonly used values should have default values inserted.

e.g.: order in checker creation.

### PR DESCRIPTION
**Pull request by hermannlallah**
closes #541

### CODE REVIEW

File: web-gui/src/app/dialogs/task-new-dialog/task-new-dialog.component.ts [hunk 1]
```diff
@@ -21,10 +24,13 @@ import {CheckerConfig} from '../../model/CheckerConfig';
   styleUrls: ['./task-new-dialog.component.scss']
 })
 export class TaskNewDialogComponent implements OnInit {
+
+
+
```
Comment (mxsph) [inline web-gui/src/app/dialogs/task-new-dialog/task-new-dialog.component.ts:29]:
```suggestion
```

File: web-gui/src/app/dialogs/task-new-dialog/task-new-dialog.component.ts [hunk 2]
```diff
@@ -12,6 +12,9 @@ import {mergeMap, map} from 'rxjs/operators';
 import {CheckerService} from '../../service/checker.service';
 import {CheckerConfig} from '../../model/CheckerConfig';
 
+let currentDateAndOneMonthLater = new Date();
+currentDateAndOneMonthLater.setMonth(currentDateAndOneMonthLater.getMonth() + 1);
```
Comment (mxsph) [inline web-gui/src/app/dialogs/task-new-dialog/task-new-dialog.component.ts:16]: Generating the date here works, but leads to the problem that it is only generated once and therefore the time does not correspond to the current time. Here you should define a function that generates and returns the date:

grafik

**Review by mxsph — CHANGES_REQUESTED** [review]
There are still some problems:

- [x] Filling in the order does not work:
  
  Peek 2022-02-07 13-43

- [x] It would be practical if the submission type 'Text' is selected by default when creating a task.

**Comment by Jonas960** [comment]
@mxsph The default values are now filled correctly. For filling the order, I took '1' as the default value. If this value is not correct, let me know

**Review by mxsph — CHANGES_REQUESTED** [review]
> @mxsph The default values are now filled correctly. For filling the order, I took '1' as the default value. If this value is not correct, let me know

- [x] Basically, `1` is the correct default value, but the order of the previous `plus 1` would make the most sense.

- [ ] If you specify a `0` as order, the frontend complains that not all fields have been filled in, this should be adjusted.

- [x] For the PR to be ready, the error from the comment above https://github.com/thm-mni-ii/feedbacksystem/pull/615#discussion_r800629457 should also be fixed.

**Review by mxsph — CHANGES_REQUESTED** [review]

**Review by mxsph — APPROVED** [review]

**Pull request merged**